.github/workflows/codeql.yml

```diff
@@ -0,0 +1,71 @@
+name: "CodeQL Advanced"
+
+on:
+  push:
+  pull_request:
+    paths-ignore:
+    - '**.md'
+  schedule:
+    - cron: '43 2 * * 5'
```

Comment:
Why is this so specific lol? 2:43 am on Friday

Reply:
Hah yeah it came with this when generated in the GitHub UI, can definitely modify or remove